samples/src/Microsoft.Azure.IIoT.App/src/Pages/Endpoints.razor

```diff
@@ -26,22 +27,31 @@
             <th>SecurityLevel</th>
             <th>EndpointActivation</th>
             <th>
-                <a href="/discoverers/">
-                    <span class="oi oi-arrow-left"></span>
-                </a>
+                @if ((DiscovererId != null) && (DiscovererId != "null")) {
```

Comment:
It would be best to have a single constant for this "root" or "all" or so string, that you reference throughout the entire project.

Reply:
done